baseframe/__init__.py

```diff
@@ -248,6 +248,12 @@ def init_app(
         app.assets.register('css_all', css_all)
         app.register_blueprint(self, static_subdomain=subdomain)
 
+        try:
+            module = __import__('baseframe-assets')
```

Comment:
Asset name should be a parameter to the function here. There's no need for `__import__` otherwise.